doc/spinel-protocol-src/spinel-tech-thread.md

```diff
@@ -98,18 +98,22 @@ Data per item is:
 
 ### PROP 91: PROP_THREAD_OFF_MESH_ROUTES
 * Type: Read-Write
-* Packed-Encoding: `A(t(6CbCb))`
+* Packed-Encoding: `A(t(6CbCbb))`
 
 Data per item is:
 
-* `6`: IPv6 Prefix
-* `C`: Prefix length, in bits
+* `6`: Route Prefix
+* `C`: Prefix length in bits
 * `b`: Stable flag
 * `C`: Other flags
```

Comment:
Updated the PR to address the https://github.com/openthread/openthread/issues/407.